observability/otel/otel-collector/otel-collector-extensions.adoc

```diff
@@ -446,18 +445,35 @@ include::snippets/technology-preview.adoc[]
         protocols:
           http: {}
     exporters:
-      otlp:
+      debug:
 
     service:
       extensions: [zpages]
       pipelines:
         traces:
           receivers: [otlp]
-          exporters: [otlp]
+          exporters: [debug]
 # ...
 ----
 
-<1> Specifies the HTTP endpoint that serves zPages. Use `localhost:` to make it available only locally, or `":"` to make it available on all network interfaces. The default is `localhost:55679`.
+<1> HTTP endpoint where zPages will be served. Default is `localhost:55679`.
+
+The {OTELOperator} will not expose this route in any way since the purpose of this extension is to be used as a way to troubleshoot. You can access the HTTP endpoint by forwarding the port:
+
+[source,terminal]
+----
+oc port-forward pod/$(oc get pod -l app.kubernetes.io/name=instance-collector -o=jsonpath='{.items[0].metadata.name}') 55679
+----
+
+Now, you can visit `http://localhost:55679/debug/servicez` in yor browser to get the zPages information.
```

Comment:
To avoid this appearing like a procedure, you could rephrase this sentence as follows:
```suggestion
To get the zPages information, you can visit `http://localhost:55679/debug/servicez` in your browser.
```